sds/src/lib.rs

```diff
@@ -7,7 +7,6 @@ mod encoding;
 mod event;
 mod match_action;
 
-#[cfg(any(test, feature = "testing", feature = "bench"))]
```

Comment:
We don't want to compile `event_json` in the lib _outside_ of testing and benchmarking - hence this feature flag.

You can pull in this dependency in sds-shared-library by adding the following line to the `[dev-dependencies]` in the Cargo.toml:
```
dd-sds = {workspace = true, features = ["bench"]}
```

Reply:
Actually I was not even sure that we don't want to build this outside of testing and benchmarking. But if this works, I'll go for this option 👍

Reply:
Thanks by the way, this works 🙇